src/Resources/Candidate.php

```diff
@@ -61,8 +63,14 @@ public static function fromArray(array $candidate): self
             $candidate['safetyRatings'] ?? [],
         );
 
+        $content = isset($candidate['content'])
+            ? Content::fromArray($candidate['content'])
+            : new Content($parts = [
+                new TextPart($candidate['content'] ?? ''),
+            ], Role::Model);
```

Comment:
Can you rewrite this part using `Content::text()` static method? I can merge it after the update.

```suggestion
            : Content::text('', Role::Model);
```